user-guide/live-traffic/live-traffic.md

```diff
@@ -218,15 +239,15 @@ Use the __Replay__ context menu option to replay previously captured sessions.
 
 ### Select
 
-Use the __Select__ context menu option to select sessions that are directly related to the currently selected session. The options are to select __parent__, __children__, and __duplicate__ sessions.
+Use the __Select__ context menu option to select sessions that are directly related to the currently selected session.  The possibilities are to choose  __parent__, __children__, and __duplicate__ sessions.
```

Comment:
Additional space is added before `The possibilities`, which shouldn't be there. Also additional space is added after choose and before `__parent__` which probably shouldn't be there as well.